src/python/espressomd/analyze.py

```diff
@@ -492,7 +492,11 @@ def energy(self):
 
         """
         observable = self.call_method("calculate_energy")
-        return self._generate_summary(observable, 1, False)
+        observable = self._generate_summary(observable, 1, False)
+        if has_features("DPD"):
+            # Remove meaningless (key, value)
+            del observable["dpd"]
+        return observable
```

Comment:
Wouldn't it be easier to make DPD stress part of the pressure calculation in the core, like in #5043? With the currently proposed changes, ESPResSo scripts written in C++ and Python disagree about the total pressure.

Reply:
That would have been fine. Other features also have zero contribution in several fields, for example the cold Drude thermostat doesn't contribute to the energy.